Integrations/Cymulate/Cymulate.yml

```diff
@@ -0,0 +1,67 @@
+category: Vulnerability Management
+commonfields:
+  id: Cymulate
+  version: -1
+configuration:
+- defaultvalue: 179743a665726a99fdee3bb0d10ba3fe
+  display: x-token
+  name: x-token
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Show only 'penatrated' incidents
+  hidden: false
+  name: onlyPenatrated
+  required: false
+  type: 8
```

Comment:
in the code you use param `module_type` but you haven't defined it in the yml. please define as a single select and define all the available options for module_type

Reply:
as we agreed with francesco - we now support only module_type="immediate_threats"